examples/text_to_image/train_text_to_image_sdxl.py

```diff
@@ -935,7 +936,10 @@ def preprocess_train(examples):
     del compute_vae_encodings_fn, compute_embeddings_fn, text_encoder_one, text_encoder_two
     del text_encoders, tokenizers, vae
     gc.collect()
-    torch.cuda.empty_cache()
+    if is_torch_npu_available():
+        torch_npu.npu.empty_cache()
+    else:
+        torch.cuda.empty_cache()
```

Comment:
```suggestion
    elif torch.cuda.is_available():
        torch.cuda.empty_cache()
```

Reply:
Thanks for your suggestion, I will change it